1-js/01-getting-started/2-code-editors/article.md

```diff
@@ -55,7 +55,7 @@ If you don't know what to choose, you can consider these ones.
 
 ## Let's not argue
 
-The editors in the lists above are those that either I or my friends who I consider good developers have been using for a long time and are happy with.
+The editors in the lists above are those that either I or my friends who consider good developers have been using for a long time and are happy with.
```

Comment:
"friends who consider good developers" - has another meaning than intended.

I mean: "I or those of my friends who I consider to be good developers"

Reply:
I see. Thanks!